ipa-core/src/report/hybrid.rs

```diff
@@ -988,10 +988,10 @@ impl UniqueTagValidator {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(test, unit_test, feature = "in-memory-infra"))]
```

Comment:
Is the `in-memory-infra`  restriction temporary, or permanent?

Reply:
I would assume permanent. [This is the errror](https://github.com/private-attribution/ipa/actions/runs/11785552671/job/32827030470#step:10:83) without it; it seems that `TestWorld` is gated behind `in-memory-infra`, unless I'm misusing that.

Reply:
trying this out with `run_random` so we don't need this.